node/lib/util/cherry_pick_util.js

```diff
@@ -743,10 +745,27 @@ exports.rewriteCommit = co.wrap(function *(repo, commit) {
 
     yield exports.closeSubs(opener, picks);
 
+    // Check whether conflicts exist
+    let conflictsExist = false;
```

Comment:
Instead of adding a new boolean, just change the "if" condition below to be:
    if (Object.keys(conflicts).length !== 0)